src/QMCHamiltonians/Listener.hpp

```diff
@@ -82,6 +83,30 @@ struct ListenerOption
   const std::vector<ListenerVector<T>>& ion_values;
 };
 
+/// Type for representing per particle energy values at the Crowd scope.
+template<typename T>
+using CrowdEnergyValues = std::unordered_map<std::string, std::vector<Vector<T>>>;
```

Comment:
This vector of Vector. Could you define what the indices? I cannot guess which is for walker and which is for particle.